source/renderer/app/components/wallet/tokens/wallet-token-picker/WalletTokenPickerCheckbox.js

```diff
@@ -0,0 +1,41 @@
+// @flow
+import React, { useCallback } from 'react';
+import { injectIntl } from 'react-intl';
+import { PopOver } from 'react-polymorph/lib/components/PopOver';
+import { Checkbox } from 'react-polymorph/lib/components/Checkbox';
+import { messages } from './WalletTokenPicker.messages';
+import { MAX_TOKENS } from './const';
+import type { ItemProps as Props } from './types';
+
+const WalletTokenPickerCheckbox = ({
+  intl,
+  className,
+  isChecked,
+  isMaxCount,
+  isDisabled,
+  uniqueId,
+  toggleCheckbox,
+}: Props) => {
+  const checked = isChecked || isDisabled;
```

Comment:
why is this check taking `isDisabled` into account?

Reply:
In this case, isDisable is true when the token has already been selected before (its already added in the send screen), so its disabled. I'll update this prop name to something more specific.